gapic/metadata/gapic_metadata.proto

```diff
@@ -0,0 +1,84 @@
+// Copyright 2020 Google LLC.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+syntax = "proto3";
+
+package google.gapic.metadata;
```

Comment:
if it's in `package google.gapic.metadata`, should it go to `google/gapic/` instead of just `gapic/`?

Reply:
I was wondering. That would be nice in order to parallel the directory structure.

OTOH, is it really necessary? We have the `gapic/` directory here, and it would be nice to not get people confused into thinking "gapic" is a Google service....

Reply:
I don't insist here - we can as well leave it here.